includes/components/class-attribute.php

```diff
@@ -392,6 +392,7 @@ public function register_attributes() {
 						'label'          => $attribute_object->post_title,
 						'display_areas'  => array_filter( (array) $attribute_object->hp_display_areas ),
 						'display_format' => (string) $attribute_object->hp_display_format,
+                        'default'        => (string) $attribute_object->hp_default,
```

Comment:
Please check WPCS setup, it should show a warning for spaces instead of tabs as indents (just tested this in VS Code). The WordPress itself and HivePress with all the extensions already uses tabs so committing spaces will cause formatting issues